langkit/templates/pkg_implementation_body_ada.mako

```diff
@@ -3704,9 +3704,12 @@ package body ${ada_lib_name}.Implementation is
       if Ent.Node /= null then
          declare
             Node_Image : constant Text_Type := Short_Text_Image (Ent.Node);
+            File_Name : constant Text_Type := To_Text (
+              Ada.Directories.Simple_Name (Get_Filename (Unit (Ent.Node))));
```

Comment:
For clarity/brevity, I suggest using the dot notation as much as possible: `Ent.Node.Unit.Get_Filename`

Reply:
Partially done, I couldn't use the dot notation for Get_Filename.

Reply:
That’s unexpected: we do use dot notations for `Get_Filename` in the code base, and it should work everywhere… what happened when you tried?

Reply:
```
/langkit/contrib/python/build/src/libpythonlang-implementation.adb:6632:58: error: no selector "Get_Filename" for type "Analysis_Unit_Type" defined at libpythonlang-implementation.ads:10751
   [Ada]          liblktlang-generic_introspection.ads
/langkit/contrib/lkt/build/src/liblktlang-implementation.adb:9803:58: error: no selector "Get_Filename" for type "Analysis_Unit_Type" defined at liblktlang-implementation.ads:16768

   compilation of liblktlang-implementation.adb failed
```

Reply:
Oh!! I did not realize we were in Langkit Mako templates… This Lkt-specific code does not belong here: `__prelude` is a Lkt-specific thing, which does not necessarily exists in other Langkit-generated frontends.

IIRC, what we discussed live was to roll your own implementation of an `Image` in `lkt_toolbox.adb`

Reply:
I remember, but I thought this was a possible solution, sorry about that, I'm going to fix it.